src/component/FilterSection.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useProduct } from "../context/product-context";
+import "../App.css";
+function FilterSection() {
+  const { state, dispatch } = useProduct();
```

Comment:
dont use generic names like state and dispatch.
Instead use names lkes productState and productDispatch.
Because you may have a lot of reducers. then you'll get confused while referring to state and dispatch of different reducers

Reply:
Thank you Piyush for susggestion made their name more specific